bucket/waterfox-current.json

```diff
@@ -0,0 +1,31 @@
+{
+    "version": "2019.10",
+    "description": "The 100% fresh, free-range, ethical browser.",
+    "homepage": "https://www.waterfox.net",
+    "license": "MPL-2.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://storage-waterfox.netdna-ssl.com/releases/win64/installer/Waterfox%20Current%202019.10%20Setup.exe#/dl.7z",
+            "hash": "d208a40ad818a526da8b8ab8cb43a084658172fea6393a785b3f7a29c00e1f1a"
+        }
+    },
+    "extract_dir": "core",
+    "bin": "waterfox.exe",
+    "shortcuts": [
+        [
+            "waterfox.exe",
+            "Waterfox"
+        ]
+    ],
+    "checkver": {
+        "url": "https://www.waterfox.net/update/win64/56.0/en-US/release/update.xml",
```

Comment:
This maybe doesn't work.

```xml
<patch type="complete" URL="https://storage-waterfox.netdna-ssl.com/releases/win64/update/waterfox-classic-2019.10.en-US.win64.complete.xz.mar" hashFunction="SHA512" hashValue="137a696c61d46783759a6661b7d406d14c5175de7bcd0c8fc759361ab76ad5a25fc15bd9ae3b5050360bfa667cbd8bf59940cce0ed6104dd6e2532108a6f2f26" size="86254889"/>
```

Reply:
Do you mean the SHA512 hashValue cannot be applied to `Waterfox Current 2019.10 Setup.exe`?  

or the update channel `https://www.waterfox.net/update/win64/56.0/en-US/release/update.xml` may not be maintained in the future?

I think `https://storage-waterfox.netdna-ssl.com/releases/win64/update/waterfox-classic-2019.10.en-US.win64.complete.xz.mar` (and the hash value) is intended to be used by the update program of Waterfox browser.

Reply:
```waterfox-classic-2019.10.en-US.win64.complete.xz.mar```

This url ckeck update for version 56.0 and return classic-2019.10, I wonder if it works for current branch and classic branch. Maybe we need to update this url?

Reply:
https://www.waterfox.net/blog/waterfox-2019.10-release-download/

Looks like the classic and current branch will be released at the same time.
However, I found there is no content at `https://www.waterfox.net/update/win64/2019.10/en-US/release/update.xml`, so it is possible that the XML channel will no longer be maintained.

Therefore, it makes more sense to check version updates from the homepage.

Reply:
> However, I found there is no content at https://www.waterfox.net/update/win64/2019.10/en-US/release/update.xml, so it is possible that the XML channel will no longer be maintained.

It's correct. Only when there is a new version there is content. I agree to change checkver because it seems that its update url is not as stable as firefox.